2022-war-in-ukraine.rst

```diff
@@ -0,0 +1,49 @@
+.. post:: March 4, 2022
+   :author: David
+   :location: SAN
+
+
+War in Ukraine and what it means for Read the Docs
+==================================================
+
+At Read the Docs, we are outraged and saddened by the invasion of Ukraine
+and we condemn this act of violence as wrong and unlawful.
+We are monitoring the situation in Europe
+and how it relates to our employees, customers, and our services to the open source world.
+
+Read the Docs is a very small company with only seven employees
+and fortunately none of them are currently in harm's way.
+Our impact, however, on the open source ecosystem is bigger than our size
+and we have users all over the world including in Ukraine and Russia.
+
+
+Sanctions
+---------
+
+We are continuing to monitor the situation with respect to sanctions.
+Read the Docs is hosted and incorporated in the United States
+and we may be compelled to take action.
+
+We aim to be a platform for all developers
+and we do not intend to prevent users from specific countries from using our platform
+especially as it relates to open source work.
+
+Read the Docs is currently partially blocked in Russia by the Russian government.
+While we are working to resolve this,
+it does not appear that this block is related to war in Ukraine.
+
+
+What we are doing and what you can do
+-------------------------------------
+
+As the situation develops, we will be transparent about any actions we take.
+We take our role of helping the open source community with documentation seriously
+and we aim to be a platform for all developers.
+
+Over the coming days, our advertising arm `EthicalAds <https://ethicalads.io>`_
+will be running a series of community ads aimed at raising money
+for the `United Nations Crisis Relief Fund <https://crisisrelief.un.org/>`_
+and other missions aimed at humanitarian aid for Ukraine.
+If you can donate to help provide humanitarian aid, please do.
+
+At Read the Docs, we are hoping that peace is restored as quickly as possible.
```

Comment:
The lead in sounded authoritative (?). Or something. I really like the sentiment as an outro though.

```suggestion
We are hoping that peace is restored as quickly as possible.
```

Feels like its missing a clause still though